charts/karma/Chart.yaml

```
@@ -6,8 +6,8 @@ home: https://github.com/prymitive/karma
 sources:
   - https://github.com/wiremind/wiremind-helm-charts/tree/main/charts/karma
   - https://github.com/prymitive/karma
-version: 2.4.0
-kubeVersion: '>= 1.19-0'
+version: 2.4.1
```

Since this is new feature, I would suggest
```suggestion
version: 2.5.0
```